src/main/java/edu/usf/cutr/gtfsrtvalidator/background/BackgroundTask.java

```diff
@@ -201,24 +202,25 @@ public void run() {
             // Use the same current time for all rules for consistency
             long currentTimeMillis = System.currentTimeMillis();
             consoleOutput = new StringBuffer();
+            timestampUtils = new TimestampUtils();
```

Comment:
It seems wasteful to instantiate a new TimestampUtils class on each new thread execute.  I'd also prefer to keep utility classes as static that aren't instantiated.  Was there a reason why the alternate solutions outlined in https://github.com/CUTR-at-USF/gtfs-realtime-validator/issues/264 wouldn't work?